schedule/yast/opensuse/agama_autoinst.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
let's create `schedule/yam/` and not differenciate yet between opensuse or sle, we can see later how it goes, so :
`schedule/yam/agama_auto`
as discussed offline, with can use just `auto` instead of `autoinst` is shorter and in the right folder is well understood IMO.